terraform/bucket_roles.tf

```diff
@@ -7,16 +7,14 @@ module "production_bucket_access" {
   }
 
   account_ids = [data.aws_caller_identity.current.account_id]
+  additional_role_tags = {
+    "GitHub_Secret_Name"             = "TEST_ROLE_TO_ASSUME",
+    "GitHub_Secret_Terraform_Lookup" = "arn"
+  }
   entity_name = aws_iam_user.user.name
   role_name   = "ThirdPartyBucketRead-${aws_iam_user.user.name}"
-  role_tags = merge(var.tags,
-    {
-      "GitHub_Secret_Name"             = "TEST_ROLE_TO_ASSUME",
-      "GitHub_Secret_Terraform_Lookup" = "arn"
-    }
-  )
-  s3_bucket  = var.production_bucket_name
-  s3_objects = var.production_objects
+  s3_bucket   = var.production_bucket_name
+  s3_objects  = var.production_objects
 }
 module "staging_bucket_access" {
```

Comment:
Does staging not need the `additional_role_tags` as in the production module?

Reply:
This is a good question that I have completely forgotten the answer to.  We should include a comment if we do indeed only need this for Production and not Staging.

Reply:
My understanding is that the additional tags being applied mark the role as being _the_ role to assume by the AWS test user for this repo before running `molecule test`.  We create both a production and a staging _policy_ because any Packer repos that use this Ansible role will attach the production or staging policy to the corresponding production or staging Packer role; the production or staging Packer role is assumed by the AWS Packer build user depending on whether a production or staging AMI is being used, respectively.

@dav3r, please make sure you agree with what I wrote here.  It's possible I have a misconception about the AWS test/build users because I didn't work with you and @felddy when you sorted out how to set those up to work in the COOL.

Reply:
@jsf9k - Yes, that all sounds correct and makes sense to me.  I vaguely recall that we chose the Production role to be used for [the `molecule test` run](https://github.com/cisagov/ansible-role-cobalt-strike/blob/develop/.github/workflows/build.yml#L113-L122) rather than Staging because Production was less likely to be disturbed or altered, which could lead to failed tests.  But I could be imagining that.  Either way, I think we should add a comment to remind our future selves what is going on here.

Reply:
Please see commit 96963ea.

Reply:
Perfect, thanks for putting in that comment!

Reply:
Perfect. Thanks for getting that documented for future us.